packages/hierarchies-react/api/presentation-hierarchies-react.api.md

```diff
@@ -233,25 +222,24 @@ export const TreeNodeRenderer: FC<PropsWithRef<TreeNodeRendererProps & RefAttrib
 
 // @alpha (undocumented)
 interface TreeNodeRendererOwnProps {
-    actions?: Array<(node: PresentationHierarchyNode) => TreeItemAction>;
     error?: ErrorNode;
+    getActions?: (node: PresentationHierarchyNode) => ReactNode[];
```

Comment:
We might as well make this a `ReactElement[]` as we expect a `Tree.ItemAction`.

Reply:
I wanted to match type of `actions` property on `Tree.Item`.

Reply:
This just feels a bit weird cause `ReactNode` already includes ReactElement[], and `ReactNode[]` is a bit ambiguous on what it actually means. But I guess its fine.

Reply:
Or both should use `ReactNode` as it's union definition also includes `Iterable<ReactNode>`.